src/main/scala/com/databricks/sparkdl/DeepImageFeaturizer.scala

```diff
@@ -147,7 +162,7 @@ object DeepImageFeaturizer extends DefaultParamsReadable[DeepImageFeaturizer] {
 
   // TODO: support batched graphs with mapBlocks
 
-  private[sparkdl] trait NamedImageModel {
+  protected[sparkdl] trait NamedImageModel {
```

Comment:
Why this change? Do we expect this to be changed in subclasses?

Reply:
I needed to access it from the Models.scala file

Reply:
I think `private[sparkdl]` means you can access it from within the `sparkdl` package.

Reply:
True, good catch